### ISSUE TITLE
Clarification on features

### ISSUE THREAD

**Issue by propertyhive** [reapit/foundations#6959]
We currently use the Foundations API to pull in properties to client's sites and have a question surrounding features (i.e. key feature bullet points you normally see on a site).

At the moment we use the' specialFeatures' field but have been told by Sikander in your team that this is wrong and that we should use the 'summary' field to obtain the bulleted features. 

After looking at a clients data though this comes through as one long string:

![67k9PXXAyE587r6UFu7ioJIJpWPKeeMe](https://user-images.githubusercontent.com/13717200/170354440-00404912-ba7c-468d-8359-a6ce267ed001.png)

.. .and sometimes has the bullet character (u2022) as above, and sometimes doesn't:

![v9nyxgppwlv8m70EJLYmZ5ASrvSCbeaw](https://user-images.githubusercontent.com/13717200/170354460-ee20370f-7e78-46bd-9e41-b33a1f18b383.png)

This, and by the generic name 'summary' suggests to me this box is almost freetype and not always used for features.

If this summary field is solely used for features can you elaborate on how I can extract these into individual features seeing as they are sent as one long string?

In summary: Can you clarify which field we should be using for key features?

Hope that makes sense but let me know if anything is unclear.

Thanks,
Steve

**Comment by AshDeeming** [comment]
Hi @propertyhive 

Where a client populates they're key features can differ based on preference. 


Some may use the Summary, Description, LongDescription or Rooms. 
specialFeatures as per the swagger docs, relates to a subsection of Attributes for a Property;
![image](https://user-images.githubusercontent.com/61687775/170450136-31310731-2eaf-4c1f-b071-b5489daf0568.png)

**Comment by AshDeeming** [comment]
Hi @propertyhive 
I'll close this ticket due to no further response - please reach out if you have any further questions.
